cms/envs/common.py

```diff
@@ -433,6 +436,13 @@
 
 ################################# Middleware ###################################
 
+# TODO: Remove Django 1.11 upgrade shim
+# SHIM: Remove birdcage references post-1.11 upgrade as it is only in place to help during that deployment
+if django.VERSION < (1, 11):
```

Comment:
Could we change all these conditions to only use birdcage for Django < 1.9?  The current version works with 1.8, but starts breaking even with 1.9 due to code moves within Django (and we want to run tests with 1.9 and 1.10).  The lack of birdcage in test runs for versions we don't plan to deploy shouldn't hurt us any.

Reply:
👍

Reply:
Sure thing